test/goose/console_test.clj

```diff
@@ -0,0 +1,35 @@
+(ns goose.console-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [goose.broker :as broker]
+            [goose.brokers.redis.console :as redis-console]
+            [goose.console :as console]
+            [goose.test-utils :as tu]
+            [ring.mock.request :as mock])
+  (:import (java.lang AbstractMethodError IllegalArgumentException)))
+
+(use-fixtures :each tu/redis-fixture tu/reset-called?-fixture)
```

Comment:
Suggested rename: `reset-stubs`. But setting up stubs and clearing them in each tests will make the tests easier to reason. I'd suggest not making it a fixture.

Reply:
Renamed to reset-stubs. 
Also, I am leaning towards using [spy](https://github.com/alexanderjamesking/spy), so the setup and teardown will happen in each test.